services/updates-svc/stories/updates_test.go

```diff
@@ -24,6 +38,61 @@ func TestOpenAndClosingReceiveUpdatesStream(t *testing.T) {
 	assert.NoError(t, stream.CloseSend())
 }
 
+func TestReceivingEvents(t *testing.T) {
+	ctx := context.Background()
+	es := eventstoredb.SetupEventStoreByEnv()
+
+	updatesClient := updatesServiceClient()
+
+	bedId := uuid.New()
+
+	req := &pb.ReceiveUpdatesRequest{}
+	stream, err := updatesClient.ReceiveUpdates(ctx, req)
+	require.NoError(t, err)
+
+	go func() {
+		// test receive one event that got emitted after subscribing
+		res, err := stream.Recv()
+		requireTrueBool(t, assert.NoError(t, err))
```

Comment:
you can't use require, but can use assert? isn't that largly the same library

Reply:
Jup. The underlying .Fail() functions behave differently in Goroutines. require.() calls .FailNow()